pos-print/src/main/java/com/clouway/pos/print/adapter/http/ReportService.kt

```diff
@@ -58,17 +57,20 @@ class ReportService(private var factory: PrinterFactory) {
     return Reply.with("Report completed").`as`(Json::class.java).ok()
   }
 
-  private fun adapt(state: String): RegisterState {
+  private fun adaptToState(state: String): RegisterState {
```

Comment:
Enums could be constructor from string values directly and gson already provides that. This will change the behaviour of the service: 

If GSON tries to parse value that is not in enum should fail, as valueOf function will fail 
If adapt function is used, a default value will be used by default if someone provides bad request. 

Currently there are no custom handlers which to return properer responses for badly formed HTTP requests (as the nature of sitebricks), so it depends.